isisd/isis_zebra.h

```diff
@@ -36,15 +36,16 @@ void isis_zebra_stop(void);
 
 struct isis_route_info;
 struct sr_prefix;
+struct sr_adjacency;
 
 void isis_zebra_route_add_route(struct prefix *prefix,
 				struct prefix_ipv6 *src_p,
 				struct isis_route_info *route_info);
 void isis_zebra_route_del_route(struct prefix *prefix,
 				struct prefix_ipv6 *src_p,
 				struct isis_route_info *route_info);
-void isis_zebra_install_prefix_sid(const struct sr_prefix *srp);
-void isis_zebra_uninstall_prefix_sid(const struct sr_prefix *srp);
+void isis_zebra_send_prefix_sid(int cmd, const struct sr_prefix *srp);
+void isis_zebra_send_adjacency_sid(int cmd, const struct sr_adjacency *sra);
```

Comment:
I'm not crazy about sticking the zapi command ids in the apis like this - was there a reason to lose the _install / _uninstall approach? or ... a 'bool add_p' to distinguish the add and delete cases?

Reply:
I would reduce the number of function expose by isis_zebra.c to isis_sr.c. Regarding the boolean value, I found this kind of variable not very explicit. I mean that you need to go to the function definition to understand the signification behind the boolean value. Here, by using ZAPI command value you exactly now what you are doing.

In addition, if there is an `install` and `uninstall` function for Prefix-SID, there is only one function for Adjacency SID. So, the modification is also here to make function consistent between Prefix-SID and Adjacency SID.

Just let me know if you prefer a `install` / `uninstall` function or usage of ZAPI cmd

Reply:
I correct all the check style in the last rebase / push force except the one about the DEFUN style which IMHO deserve the way we usually used DEFUN

Reply:
well, no I don't want to hold this up any longer, let's move forward.